### ISSUE TITLE
Support render property of block.json when using useBlockAssets

### ISSUE THREAD

**Issue by JakePT** [10up/10up-toolkit#299]
### Is your enhancement related to a problem? Please describe.

As of WordPress 6.1.0, block.json supports a [`render`](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#render) property, which can point to a PHP file that will be used to render the block:

```json
{
    "render": "file:./render.php"
}
```

Currently `useBlockAssets` is not aware of this property, and will not copy the render file to the `dist/` directory. A workaround is to use the relative path back to the original file:

```json
{
    "render": "file:../../../includes/blocks/block/render.php"
}
```

Ideally the render file would be copied so that the shorter relative path could be used, as with the other supported properties, such as `editorScript`.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fabiankaegy** [comment]
Thank you for reporting this! Indeed this is an issue we want to address. Currently we hadn't run into this because on most of our projects we had the best practice of naming the render file `markup.php` and because of that we only added that name in the list of files that get moved over: https://github.com/10up/10up-toolkit/blob/a640a663ad5d559a59fc5031a6ab9fa5d018e996/packages/toolkit/config/webpack/plugins.js#L170

We should change that to be a wildcard that moves any PHP files inside the block folder over to the dist folder.

**Comment by nicholasio** [comment]
Closing as this has been released in 5.2.1
